website/docs/r/project.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_project"
+description: |-
+  Manages project.
+subcategory: "Projects API Specification"
+---
+
+# ibm_project
+
+Provides a resource for project. This allows project to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_project" "project_instance" {
+  configs {
+		id = "id"
+		name = "name"
+		labels = [ "labels" ]
+		description = "description"
+		locator_id = "locator_id"
```

Comment:
kindly provide working example
kkindly provide valid values to the argument to make this doc more user friendly.